src/main/java/com/blamejared/ambientenvironment/AmbientEnvironment.java

```diff
@@ -16,96 +16,98 @@
 import org.apache.commons.lang3.tuple.Pair;
 
 @Mod("ambientenvironment")
-public class AmbientEnvironment {
+public final class AmbientEnvironment {
 
     public AmbientEnvironment() {
         ModLoadingContext.get().registerExtensionPoint(ExtensionPoint.DISPLAYTEST, () -> Pair.of(() -> FMLNetworkConstants.IGNORESERVERONLY, (a, b) -> true));
         FMLJavaModLoadingContext.get().getModEventBus().addListener(this::doClientStuff);
     }
 
     private void doClientStuff(final FMLClientSetupEvent event) {
-        ColorResolver grassColor = BiomeColors.GRASS_COLOR;
-        ColorResolver waterColor = BiomeColors.WATER_COLOR;
-//        BiomeColors.IColorResolver foliageColor = BiomeColors.FOLIAGE_COLOR;
+        final ColorResolver
+                grassColor = BiomeColors.GRASS_COLOR,
+                waterColor = BiomeColors.WATER_COLOR;
+//        final BiomeColors.IColorResolver foliageColor = BiomeColors.FOLIAGE_COLOR;
 
-        int levels = 2;
-        PerlinNoiseGenerator NOISE_GRASS = new PerlinNoiseGenerator(new SharedSeedRandom("NOISE_GRASS".hashCode()), IntStream.rangeClosed(0, levels));
-        PerlinNoiseGenerator NOISE_WATER = new PerlinNoiseGenerator(new SharedSeedRandom("NOISE_WATER".hashCode()), IntStream.rangeClosed(0, levels));
-//        PerlinNoiseGenerator NOISE_FOLIAGE = new PerlinNoiseGenerator(new Random("NOISE_FOLIAGE".hashCode()), levels);
+        final int levels = 2;
+        final PerlinNoiseGenerator
+                NOISE_GRASS = new PerlinNoiseGenerator(new SharedSeedRandom("NOISE_GRASS".hashCode()), IntStream.rangeClosed(0, levels)),
+                NOISE_WATER = new PerlinNoiseGenerator(new SharedSeedRandom("NOISE_WATER".hashCode()), IntStream.rangeClosed(0, levels));
```

Comment:
Please can you change this back to how it was before, making it a single declaration like this doesn't help anything and just makes the code harder to read.